atomone/chain.json

```diff
@@ -0,0 +1,141 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "atomone",
+  "chain_type": "cosmos",
+  "chain_id": "atomone-1",
+  "website": "https://atom.one",
+  "pretty_name": "AtomOne",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "atone",
```

Comment:
Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
It's already registered https://github.com/satoshilabs/slips/pull/1803

Reply:
oh damned there's a typo, let me fix that.